spec.html

```diff
@@ -10440,6 +10440,25 @@ <h1>Static Semantics: CodePointAt ( _string_, _position_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-stringindexof">
```

Comment:
As a note, I'd love to just click "commit suggestion" instead of applying all these changes locally but GitHub won't let me, instead showing the "This diff has recently been updated. Refresh and try again." error. If there is some trick to making this work, please let me know.

Reply:
It appears to be a [known bug](https://github.community/t5/How-to-use-Git-and-GitHub/Cannot-apply-suggestion-in-PR-as-collaborator-with-Write-access/m-p/45917)

I considered making a PR against your branch, but the number of edits didn't seem quite big enough to warrant the bother.